src/test/java/io/vlingo/actors/plugin/mailbox/telemetry/DefaultMailboxTelemetryTest.java

```diff
@@ -42,14 +39,20 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void testThatSendRegistersACounterOnTheActorsMailbox() {
+  public void testThatSendAndPullRegistersACounterOnTheActorsMailbox() {
     telemetry.onSendMessage(message);
+    assertGaugesForMessageAre(1, 1);
 
+    telemetry.onPulledMessage(message);
+    assertGaugesForMessageAre(0, 0);
+  }
+
+  private void assertGaugesForMessageAre(final int expectedPerActor, final int expectedGlobal) {
     AtomicInteger lag = telemetry.gauges().get("RandomActor."+addressOfActor);
-    assertEquals(lag.get(), 1);
+    assertEquals(expectedPerActor, lag.get());
```

Comment:
Have we agreed that "lag" will change to "pending" or some other term? What does "lag" mean specifically in this context?